client/rest-high-level/src/main/java/org/elasticsearch/client/ml/datafeed/DatafeedConfig.java

```diff
@@ -108,9 +113,16 @@ private static BytesReference parseBytes(XContentParser parser) throws IOExcepti
     private final Integer scrollSize;
     private final ChunkingConfig chunkingConfig;
 
+    /**
+     * The window of time to check for missing data
+     */
+    private final TimeValue delayedDataCheckWindow;
+    private final boolean shouldRunDelayedDataCheck;
```

Comment:
This should be `Boolean` to allow null values so that the server side can default.